eventtracking/backends/xapi.py

```diff
@@ -0,0 +1,17 @@
+import logging
+from celery.utils.log import get_task_logger
+
+from eventtracking.backends.routing import RoutingBackend
+
+LOGGER = get_task_logger(__name__)
+
+
+class XApiBackend(RoutingBackend):
```

Comment:
Have you been able to figure out why we need to inherit `xapi` and `caliper` backends from `RoutingBackend`?

Reply:
I've replied about that to the comments above.